### ISSUE TITLE
Layout/IndentationWidth conflicts with Layout/AccessModifierIndentation EnforcedStyle: outdent

### ISSUE THREAD

**Issue by toy** [rubocop-hq/rubocop#6861]
Setting `EnforcedStyle: outdent` for `Layout/AccessModifierIndentation` causes conflict with `Layout/IndentationWidth` starting with rubocop 0.66.0.

## Expected behavior

No error as there is no new cops relevant to the example file.
Running same commands using previous rubocop version `rubocop _0.65.0_` and `rubocop _0.65.0_ --auto-correct` finishes successfully and without any corrections.

## Actual behavior

Running `rubocop` reports an error, running `rubocop --auto-correct` reports two corrected errors then fails with message about infinite loop + backtrace.

## Steps to reproduce the problem

Create file `test.rb`:
```rb
# frozen_string_literal: true

# Test Layout/IndentationWidth problem
class Test
private

  def test
    :test
  end
end
```

Create file `.rubocop.yml`:
```yml
Layout/AccessModifierIndentation:
  EnforcedStyle: outdent
```

Run `rubocop`:
```
Inspecting 1 file
C

Offenses:

test.rb:5:1: C: Layout/IndentationWidth: Use 2 (not 0) spaces for indentation. (https://github.com/rubocop-hq/ruby-style-guide#spaces-indentation)
private


1 file inspected, 1 offense detected
```

Run `rubocop --auto-correct`:
```
Inspecting 1 file
C

Offenses:

test.rb:5:1: C: [Corrected] Layout/IndentationWidth: Use 2 (not 0) spaces for indentation. (https://github.com/rubocop-hq/ruby-style-guide#spaces-indentation)
private

test.rb:5:3: C: [Corrected] Layout/AccessModifierIndentation: Outdent access modifiers like private. (https://github.com/rubocop-hq/ruby-style-guide#indent-public-private-protected)
  private
  ^^^^^^^

0 files inspected, 2 offenses detected, 2 offenses corrected
Infinite loop detected in …/test.rb.
…/lib/ruby/gems/2.5.0/gems/rubocop-0.66.0/lib/rubocop/runner.rb:263:in `check_for_infinite_loop'
…/lib/ruby/gems/2.5.0/gems/rubocop-0.66.0/lib/rubocop/runner.rb:246:in `block in iterate_until_no_changes'
…/lib/ruby/gems/2.5.0/gems/rubocop-0.66.0/lib/rubocop/runner.rb:245:in `loop'
…/lib/ruby/gems/2.5.0/gems/rubocop-0.66.0/lib/rubocop/runner.rb:245:in `iterate_until_no_changes'
…/lib/ruby/gems/2.5.0/gems/rubocop-0.66.0/lib/rubocop/runner.rb:216:in `do_inspection_loop'
…/lib/ruby/gems/2.5.0/gems/rubocop-0.66.0/lib/rubocop/runner.rb:119:in `block in file_offenses'
…/lib/ruby/gems/2.5.0/gems/rubocop-0.66.0/lib/rubocop/runner.rb:137:in `file_offense_cache'
…/lib/ruby/gems/2.5.0/gems/rubocop-0.66.0/lib/rubocop/runner.rb:117:in `file_offenses'
…/lib/ruby/gems/2.5.0/gems/rubocop-0.66.0/lib/rubocop/runner.rb:105:in `process_file'
…/lib/ruby/gems/2.5.0/gems/rubocop-0.66.0/lib/rubocop/runner.rb:82:in `block in each_inspected_file'
…/lib/ruby/gems/2.5.0/gems/rubocop-0.66.0/lib/rubocop/runner.rb:81:in `each'
…/lib/ruby/gems/2.5.0/gems/rubocop-0.66.0/lib/rubocop/runner.rb:81:in `reduce'
…/lib/ruby/gems/2.5.0/gems/rubocop-0.66.0/lib/rubocop/runner.rb:81:in `each_inspected_file'
…/lib/ruby/gems/2.5.0/gems/rubocop-0.66.0/lib/rubocop/runner.rb:73:in `inspect_files'
…/lib/ruby/gems/2.5.0/gems/rubocop-0.66.0/lib/rubocop/runner.rb:39:in `run'
…/lib/ruby/gems/2.5.0/gems/rubocop-0.66.0/lib/rubocop/cli.rb:174:in `execute_runner'
…/lib/ruby/gems/2.5.0/gems/rubocop-0.66.0/lib/rubocop/cli.rb:75:in `execute_runners'
…/lib/ruby/gems/2.5.0/gems/rubocop-0.66.0/lib/rubocop/cli.rb:47:in `run'
…/lib/ruby/gems/2.5.0/gems/rubocop-0.66.0/exe/rubocop:13:in `block in <top (required)>'
…/lib/ruby/2.5.0/benchmark.rb:308:in `realtime'
…/lib/ruby/gems/2.5.0/gems/rubocop-0.66.0/exe/rubocop:12:in `<top (required)>'
…/bin/rubocop:23:in `load'
…/bin/rubocop:23:in `<main>'
…/bin/ruby_executable_hooks:15:in `eval'
…/bin/ruby_executable_hooks:15:in `<main>'
```

## RuboCop version

`0.66.0 (using Parser 2.6.2.0, running on ruby 2.5.3 x86_64-darwin18)`

**Comment by koic** [comment]
Thanks for the feedback. I opened a PR #6866.

**Comment by chucklor** [comment]
Any updates on this? I'm interested in having 4 spaces indent as well. I wonder if I can simply override it in the .rubocop.yml file. Thank you!

**Comment by mvastola** [comment]
Second this issue. Seems to only be a problem when the access modifier is the first line of code in the class/module.
